src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -552,6 +553,12 @@ public static void VerifyWithRevocation()
                         break;
                     }
 
+                    if (i > 3)
+                    {
+                        // Network congestion?
+                        Thread.Sleep(1000);
+                    }
```

Comment:
The sleep should really be after the logging loop, so that the "what happened" logs have a closer timestamp to "when they tried it".  Alternatively, before the call to onlineChain.Build.  But right here is pretty weird :)